deployment/v3/utils/data-archive/install.sh

```
@@ -17,12 +17,11 @@ function installing_data-archive() {
   helm repo add mosip https://mosip.github.io/mosip-helm
   helm repo update
 
-  read -p "Is values.yaml for data-archive chart set correctly as part of Pre-requisites?(Y/n) " yn;
-  if [ "$yn" != "Y" ]; then
-    echo "ERROR: values.yaml not set correctly; EXITING;";
-    exit 1;
+  read -p "Is values.yaml for data-archive chart set correctly as part of Pre-requisites?(Y/n) " yn
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)